devnet-sdk/devstack/dsl/sequencer.go

```diff
@@ -0,0 +1,22 @@
+package dsl
+
+import (
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/stack"
+)
+
+type Sequencer struct {
+	commonImpl
+
+	stack.Sequencer
```

Comment:
I prefer embedding the `stack` type into the `dsl` type in order to avoid multiple `Escape()` calls within tests. We use both `stack` APIs and sugar-DSL methods multiple times.

In any case, depending on which way we go forward, I will refactor this to be consistent with the rest of the DSL types.

Reply:
`Escape` is exactly meant to make every reach into the inner thing explicit, since we like to have DSL for all things. By making it explicit we can easily find which tests use `Escape()`, and then work on improving those tests with new DSL.